### PR TITLE
Create new bounds check test

### PR DESCRIPTION
**Pull request by adammartins**
## Description

New filter tests to check for number of values within specified range.

## Definition of Done

We can now test for values outside of required ranges!

### CODE REVIEW

**Review by samanthaibaker — APPROVED** [review]
Looks all good from here!

**Comment by breogangomez** [comment]
Hi Adam, have you run ctest for your change? Do all tests pass?

**Comment by adammartins** [comment]
Yes, ctest showed all passed.

File: src/ufo/filters/CMakeLists.txt [hunk 1]
```diff
@@ -4,6 +4,8 @@
 # which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
 
 set ( filters_files
+      PracticalBoundsCheck.cc
```
Comment (breogangomez) [inline src/ufo/filters/CMakeLists.txt:7]: I would swap these two lines, but that is OK for now

**Review by breogangomez — APPROVED** [review]

**Pull request closed without merging**